tools/tdbg/dlq_service.go

```diff
@@ -76,15 +79,15 @@ func getDLQV2Service(
 	if !ok {
 		return nil, fmt.Errorf("unknown dlq category %v", dlqType)
 	}
-	sourceCluster := c.String(FlagCluster)
-	if len(sourceCluster) == 0 {
-		return nil, fmt.Errorf("--%s is not provided", FlagCluster)
-	}
 	targetCluster, service, err := getTargetCluster(c, clientFactory)
 	if err != nil {
 		return service, err
 	}
-	return NewDLQV2Service(category, sourceCluster, targetCluster, clientFactory), nil
+	sourceCluster := c.String(FlagCluster)
+	if len(sourceCluster) == 0 {
+		sourceCluster = targetCluster
+	}
```

Comment:
nit: required for the replication queue?

Reply:
Good idea, added validation and a test for it.